lib/service_actor/playable.rb

```diff
@@ -56,7 +56,7 @@ def define_alias_input(actor, new, original)
   module PrependedMethods
     def call
       self.class.play_actors.each do |options|
-        next if options[:if] && !options[:if].call(result)
+        next unless valid_condition(options)
```

Comment:
I wonder if `valid_` is the best prefix, feel free to challenge it

Reply:
Good call. I also think that adding `actor` in its name can help against method collision. What do you think about `callable_actor?`.